nerdlandbot/commands/GuildData.py

```diff
@@ -19,6 +19,9 @@ class GuildData:
     youtube_channels: dict
     purgers: dict
     culture: str
+    pets: dict
+    pets_last_id: int
```

Comment:
```suggestion
    pets_last_id: Optional[int]
```

Or if annotation evaluation is postponed [PEP563](https://www.python.org/dev/peps/pep-0563/):

```suggestion
    pets_last_id: int | None
```